context.go

```diff
@@ -336,38 +336,62 @@ func (c *Context) Rem(d, x, y *Decimal) (Condition, error) {
 }
 
 // Sqrt sets d to the square root of x.
+// Sqrt uses the Babylonian method for computing the square root, which uses
```

Comment:
I really like this basic description of the algorithm and the big O for the number of steps. However I'm going to leave the reference as is to be consistent with the other references, none of which show up in the godoc. If you think it should be in the godoc, we can have that discussion and move them all at the same time.